### PR TITLE
Update web UI search content for 6.1

### LINKED ISSUE #2750: Web UI docs updates needed for 6.1
### Description
6.1 web UI updates:
- Everything is a field selector. You just need to ID the field and add the search term. For example, instead of `fieldSelector:linux in event.entity.subscriptions` use `linux in event.entity.subscriptions`.
- For label selectors, just use `label` instead of `labelSelector`. For example, instead of `labelSelector:region matches "us"` use `label:region matches "us"`. Remove fieldSelector and labelSelector from searches reference.
- Quick search: just type a term into each page in the web UI and the automatically programmed logic will identify the field to search. Logic is different depending on the page -- don't need to explain, just give an example or two.
- Saved searches: formats used in pre-6.1 releases will continue to work for 6.1. Maybe only need to document this in the release notes?

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/6.1/web-ui/filter/
https://docs.sensu.io/sensu-go/6.1/web-ui/searches/

### Corrections needed
Update docs as need to reflect 6.1 changes.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Updates 6.1 documentation to reflect web UI search improvements:
- Renames web-ui/filter to web-ui/search and changes terminology from filtering and selectors to search and search terms. Adds quick search description. Updates all explanations and examples to follow the improved search syntax for 6.1. Removes explanations that no longer apply after 6.1 improvements.
- Renames web-ui/searches to web-ui/searches-reference and removes fieldSelector and labelSelector description tables.
- Adds redirects in static.json for web-ui/filter and web-ui/searches.
- Updates links to web-ui/filter and web-ui/searches throughout other docs.
- Adds updated image of federated web UI landing page.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2750

### CODE REVIEW

File: content/sensu-go/6.1/web-ui/search.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: web-ui
+---
+
+The Sensu web UI includes basic search functions you can use to build customized views of your Sensu resources.
+Sensu also supports advanced web UI searches based on a wider range of resource attributes and custom labels as a [commercial feature][1].
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md:16]: Is this still true? If the web UI is now just part of the commercial package, do we just want a note at the top of every web UI page that matches the notes we use for other commercial features?

For example:
`COMMERCIAL FEATURE: Access [X FEATURE] in the packaged Sensu Go distribution. For more information, see Get started with commercial features.`

File: content/sensu-go/6.1/web-ui/search.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+{{% notice note %}}
+**NOTE**: You do not need to specify a resource type in web UI search because you must navigate to the resource page *before* you construct the search.
+{{% /notice %}}
+
+## Create advanced searches
+
+**COMMERCIAL FEATURE**: Access advanced search functions in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][1].
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md:64]: See my previous comment about commercial features.

File: content/sensu-go/6.1/web-ui/search.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+{{< code text >}}
+windows in event.check.subscriptions && event.check.handlers matches email
+{{< /code >}}
+
+## Save a search
+
+**COMMERCIAL FEATURE**: Access saved searches in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][1].
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md:179]: Same thing here re: comment about commercial features.

File: content/sensu-go/6.1/web-ui/search.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+The web UI quick search allows you to query Sensu resources without using search syntax.
+Type your search term into the search field on any page of the web UI and press `Enter`.
+Sensu will auto-complete a simple search statement for the resources on that page using substring matching.
+
+For example, on the Events page in the web UI, if you type `mysql` into the search field, Sensu will auto-complete the search statement to `event.check.name matches "mysql"`.
+
+## Create basic searches
+
+Sensu includes basic search functions for all resource pages.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md]:
```suggestion
Sensu includes these basic search functions:

- **Events page**: search by entity, check, status, and silenced/unsilenced.
- **Entities page**: search by entity class and subscription.
- **Silences page**: search by check and subscription.
- **Checks page**: search by subscription and published/unpublished.
- **Handlers page**: search by handler type.
- **Filters page**: search by action.

```

File: content/sensu-go/6.1/web-ui/search.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+
+Sensu includes these basic search functions:
+
+- **Events page**: search by entity, check, status, and silenced/unsilenced.
+- **Entities page**: search by entity class and subscription.
+- **Silences page**: search by check and subscription.
+- **Checks page**: search by subscription and published/unpublished.
+- **Handlers page**: search by handler type.
+- **Filters page**: search by action.
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md:54]:
```suggestion
```

File: content/sensu-go/6.1/web-ui/search.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+If you are using the [basic web UI search functions][5], you can create a search by clicking in the search bar at the top of the web UI page:
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/web-ui/search.md:56]: I need to understand this better -- there are many more search functions listed when I just click in the search bar, but I think I'm using a licensed version. Are these additional search functions turned off at 100 entities, or are they available without a license too?

Comment (hillaryfraley) [reply content/sensu-go/6.1/web-ui/search.md:56]: Cameron suggests a single commercial note on the page that explains this page describes the web UI that's available in our commercial distribution. If you compile from OSS, you may not see all of these features.

It's also possible that a note is not necessary, or that this information should be included at https://docs.sensu.io/sensu-go/latest/platforms/#build-from-source instead.

**Review by dabria — APPROVED** [review]
Looks great to me! Good work!

**Pull request merged**